mne/preprocessing/nirs/tests/test_nirs.py

```diff
@@ -76,7 +143,48 @@ def test_fnirs_spread_bads(fname):
     # Test spreading multiple bads and on chroma data
     raw = beer_lambert_law(raw)
     raw.info['bads'] = [raw.ch_names[x] for x in [1, 8]]
-    print(raw.info['bads'])
     raw = _fnirs_spread_bads(raw)
-    print(raw.info['bads'])
     assert raw.info['bads'] == [raw.ch_names[x] for x in [0, 1, 8, 9]]
+
+
+@testing.requires_testing_data
+@pytest.mark.parametrize('fname', ([fname_nirx_15_2_short, fname_nirx_15_2,
+                                    fname_nirx_15_0]))
+def test_fnirs_channel_naming_and_order_readers(fname):
```

Comment:
Here I am adding testing of an existing function before new functionality that was proposed in #9141 is added in a yet to be opened PR.